Projects/SEC/SEC04/SEC04-AWS100.md

```diff
@@ -0,0 +1,54 @@
+
+# SEC04-AWS100 — Create an IAM user
+
+## Cloud Service Provider
+
+* Amazon Web Services
+
+## Difficulty
+Level 100 (Introductory)
+
+## Estimated time:
+ 1 hour
+
+## Project's Author(s)
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
```

Comment:
- "Admin group" does not provide enough information for a Lvl100 project. Please describe this in detail, first time IAM user creation is not a walk in the park (as much as it seems like so to us now).
- I'd recommend creating three bullet points in the form of:
  - Create a new Group
  - Add 'AdministratorAccess' privileges to the newly created group
  - Add new IAM user to that group

I see there are link provided down below, but it's still not easy to understand that these resources have to be created seperately.